cob_monitoring/src/battery_monitor.py

```diff
@@ -78,6 +85,22 @@ def __init__(self):
         self.interval_duration_error = rospy.get_param("~interval_duration_error", 15.0)
         self.interval_duration_critical = rospy.get_param("~interval_duration_critical", 5.0)
 
+        if self.enable_horn:
+            self.set_horn_service = rospy.get_param("~set_horn_service")
+            self.horn_frequency = rospy.get_param("~horn_frequency")
+            self.horn_pulses_warning = rospy.get_param("~horn_pulses_warning")
+            self.horn_pulses_error = rospy.get_param("~horn_pulses_error")
+            self.horn_pulses_critical = rospy.get_param("~horn_pulses_critical")
+            self.horn_pulses = self.horn_pulses_critical    #default value
+            self.horn_timeout = rospy.get_param("~horn_timeout")
```

Comment:
can you please also add some default values here?

Reply:
Done!